loopchain/peer/consensus_siever.py

```diff
@@ -135,36 +157,34 @@ def __build_candidate_block(self, block_builder, next_leader, vote_result):
                                  f"peer_id({ChannelProperty().peer_id})")
                 await ObjectManager().channel_service.reset_leader(next_leader.hex_hx())
             else:
-                self.__block_generation_timer.call()
-
-    def count_votes(self, block_hash: Hash32):
-        # count votes
-        vote = self._block_manager.candidate_blocks.get_vote(block_hash)
-        if not vote.get_result(block_hash.hex(), conf.VOTING_RATIO):
-            return True  # vote not complete yet
-
-        self.__stop_broadcast_send_unconfirmed_block_timer()
-
-    # async def _wait_for_voting(self, candidate_block: 'Block'):
-    #     while True:
-    #         result = self._block_manager.candidate_blocks.get_vote_result(candidate_block.header.hash)
-    #         if result:
-    #             return True
-    #
-    #         timeout_timestamp = candidate_block.header.timestamp + conf.BLOCK_VOTE_TIMEOUT * 1_000_000
-    #         timeout = -util.diff_in_seconds(timeout_timestamp)
-    #         try:
-    #             if timeout < 0:
-    #                 raise asyncio.TimeoutError
-    #
-    #             vote_result = await asyncio.wait_for(self._vote_queue.get(), timeout=timeout)
-    #             if vote_result is None:  # sentinel
-    #                 return False
-    #
-    #         except asyncio.TimeoutError:
-    #             logging.warning("Timed Out Block not confirmed duration: " +
-    #                             str(util.diff_in_seconds(candidate_block.header.timestamp)))
-    #             return False
+                if not conf.ALLOW_MAKE_EMPTY_BLOCK:
+                    self.__block_generation_timer.call_ignore_slot()
```

Comment:
How about changing the method name?
For example `call_instantly`, `call_immediately`

Reply:
Or just call `await consensus()`